pkg/chain/ethereum/ethutil/subscribe_opts.go

```diff
@@ -0,0 +1,36 @@
+package ethutil
+
+import "time"
+
+const (
+	// DefaultSubscribeOptsTickDuration is the default duration with which
```

Comment:
I think I'll change it just do `DefaultSubscribeOptsTick`, this would be consistent with `func Tick(d Duration) <-chan Time` function from Go `time` package.

Reply:
Please see 552f2ba2234eebf836e10b5037c3b919def0ab40.